packages/react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -513,6 +516,12 @@ export interface IColumn {
    */
   sortDescendingAriaLabel?: string;
 
+  /**
+   * Accessible label for indicating that the list could be sorted by this column but isn't currently.
+   * This will be read after the main column header label.
+   */
+  sortableAriaLabel?: string;
```

Comment:
I don't see this PR rendering this anywhere. Did I miss something?

Reply:
Correct, and good catch! I'd missed where the actual aria-label was being set.